.github/workflows/release.yml

```diff
@@ -57,7 +57,9 @@ jobs:
       - name: "Store version numbers in env variables"
         run: |
           echo RELEASE_VERSION=${{ inputs.version }} >> $GITHUB_ENV
+          echo RELEASE_VERSION_WITHOUT_STABILITY=$(echo ${{ inputs.version }} | awk -F- '{print $1}') >> $GITHUB_ENV
```

Comment:
This is necessary as we need to check for a patch release. The best way is to cut the suffix (e.g. `-RC0`) from the version and only keep the major, minor, and patch numbers.

Reply:
Noted that this is using the `-F sepstring` syntax from [awk(1p)](https://man7.org/linux/man-pages/man1/awk.1p.html). It wasn't immediately clear to me due to the omitted space, but https://askubuntu.com/a/342850 helped clarify.